### ISSUE TITLE
Missing EMI entry, unexpected Hose Pulley behavior

### ISSUE THREAD

**Issue by JonasDoe** [EnigmaticaModpacks/Enigmatica9#799]
### Modpack Version

16.0

### Describe your issue.

Since I don't want to spam the issues with minor stuff I've stumbled upon, I decided to create a short list this time:
- [ ] EMI: It would be great if Ancient Knowledge fluid would be listed here. It's hard to find its receipe starting from here. Even Ancient Knowledge Potions don't have any receipe in EMI.
- [ ] When a Create Hose Pulley creates a lake of Molten Plastic, it won't cool down until I enforce a check on the fluid (e.g. by scooping one bucket). Here go's my "omg I'm so smart solution". :( Dunno, maybe that's intended, but I figure achieving the same with Routers isn't far away anyway.

If you don't want these kind of collected issues, I can restrain from it.

### Have you modified the modpack?

No

### Did the issue happen in singleplayer or on a server?

Server

**Comment by MuteTiefling** [comment]
Thanks for the report! I'd prefer keeping things separate in the future, but it's not a big deal.

For potions; there are no potion fluids in EMI. IE has buckets, but otherwise that's it. And for good reason. We have Hexerei, Immersive Engineering, Thermal Series, and Create that all inject their own unique potion fluid. At over 250 potion types times 4 different potion fluids... yeah we don't need that kind of clutter.

Potions are all crafted the same anyway, you can simply base yourself off of the vanilla brewing recipe. And for things like Mage Bloom, you can trace it back from the magebloom seed recipe: 

![image](https://github.com/EnigmaticaModpacks/Enigmatica9/assets/9543430/724b95b5-a70d-4601-ae04-ecbbae23a2e2)

Hover over the fluid there and click R to get the recipe. Since your goal is the seed anyway, it should be very natural to look up the seed and work backwards.

![image](https://github.com/EnigmaticaModpacks/Enigmatica9/assets/9543430/55db9bbb-ae32-48f6-a910-63e40a7a34ab)



For Create, I'm afraid I can't reproduce it. When I hose pully liquid plastic out into the world, it cools into plastic without issue. There are a variety of other methods you could use, however. Fluid Outlets, Liquid Hoppers with Dispenser upgrades, or Create pipes all work to place the plastic in world for cooling. 

![image](https://github.com/EnigmaticaModpacks/Enigmatica9/assets/9543430/26a1bdee-ffac-4a30-a08a-72cf7233c14f)

**Comment by JonasDoe** [comment]
Hm, I start to see the disadvantage for bundled issues. :)
Regarding the Hose Pulley: that's weird - I tested it again in Creative Mode, both in the Overwold as well as in the Twilight Forest:
https://i.imgur.com/0irQ9li.png
https://i.imgur.com/6lPzS2V.png
I waited for 5 minutes, and nothing happened. Any updates on the fluid will trigger the cooling, though, like removing a block next to it, removing a bucket if it's a "lake" etc. But yeah, you're probably right that there are enough alternatives! They have some minor disadvantages (like requiring higher tech or stress), but that's not dramatic ofc.

Regarding the potions I see your point - thanks for the remarks! :)
